packages/terra-popup/translations/en.json

```diff
@@ -1,3 +1,4 @@
 {
-  "Terra.popup.header.close": "Close"
+  "Terra.popup.header.close": "Close",
+  "Terra.popup.escapeInstruction": "Press escape to close"
```

Comment:
I think we should not add this message on component. instead we should allow consumer to pass in the value relative to the content that is being rendered through popup.

`Press escape to close` seems to be incomplete information does not provides detailed information of what can be closed on escape. Similar changes can be seen on other terra controls as well where we have add header title as part of navigation instruction to provide better screen reader experience.

I think just adding a first class prop for popup and adding it over the hookshot content should meet the expectation of UXPLATFORM-9605

Reply:
As discussed, updated the changes.